src/Fable.SignalR.DotNet/HubConnection.fs

```diff
@@ -8,7 +8,7 @@ open System.Threading
 open System.Threading.Tasks
 
 type internal IHubConnection<'ClientApi,'ClientStreamFromApi,'ClientStreamToApi,'ServerApi,'ServerStreamApi> (hub: HubConnection) =
-    member _.ConnectionId = 
```

Comment:
Ugh, so much trailing whitespace removed on account of an extension. Hopefully that's OK (can ignore whitespace in diff settings to remove this noise)